e2e/node/e2e.test.ts

```diff
@@ -519,6 +520,52 @@ describe(`End-to-end access grant tests for environment [${environment}}]`, () =
       expect(updatedDatasetTtl).toBe(updatedDatasetAsOwnerTtl);
     });
 
+    it("can use the saveSolidDatasetInContainer API for an existing dataset", async () => {
+      // Need to delete file already created in test setup to start with empty container.
+      await solidClient.deleteFile(testFileIri, {
+        fetch: resourceOwnerSession.fetch,
+      });
+
+      const testDataset = await solidClient.createSolidDataset();
+      const savedDataset = await saveSolidDatasetInContainer(
+        testContainerIri,
+        testDataset,
+        accessGrant,
+
+        {
+          fetch: requestorSession.fetch,
+          slugSuggestion: testFileName,
+        }
+      );
+
+      const datasetInPodAsResourceOwner = await solidClient.getSolidDataset(
+        testFileIri,
```

Comment:
Arguably this should be `getSourceIri(savedDataset)` as the `saveSolidDatasetInContainer` method can result in an IRI other than the `containerUri + slugSuggestion` value.